### DISCUSSION TITLE
Feature Request: Responsive Classic AJAX Add to Cart from Listing Page

### DISCUSSION THREAD

**Discussion by scottcwilson** [zencart/zencart#5778] [Ideas]
Suppose you have  DISPLAY_CART = 'false' and you're on a listing page with many products.

If you have PRODUCT_LISTING_MULTIPLE_ADD_TO_CART set to 'true', you have to set all the quantities you want on various products, then press enter, then you go back to the top of the page. 

If you have PRODUCT_LISTING_MULTIPLE_ADD_TO_CART set to 'false',  you get an Add to Cart button for each product, but when you press it, you go the product info page. 

If we had AJAX add to cart built into responsive_classic, we could allow people to stay in place on a listing page as they added products.  This would be a nice feature.

**Comment by wiztechinc** [comment]
Definitely have had this requested in the past by users and see major benefits to it. I force default as add multiple because adding one at a time is ridiculous but have definitely had folks complain about the multiples. Sales is all about reducing the number of clicks and making actual purchases easier. The faster it can happen the more sales happen.  Then I've also worked on carts where the actual product page is actually redundant - totally unnecessary or less used. Such as an industrial supplier selling drywalls screws where the pic and the measurement is all that's necessary.

**Comment by drbyte** [comment]
What about in the case of attributes?

**Comment by scottcwilson** [comment]
The add to cart button only appears if the product has no attributes (or only defaultable attributes in Zen Cart 1.5.7 and above).   So no worries there.

**Comment by scottcwilson** [comment]
For future post 1.5.7.

**Comment by drbyte** [comment]
@lat9 would an AJAX Add To Cart implementation (not necessarily a current plugin) have any negative impact on OPC?

**Comment by lat9** [comment]
> 
> 
> @lat9 would an AJAX Add To Cart implementation (not necessarily a current plugin) have any negative impact on OPC?

Given that OPC 'activates' during checkout, I don't see that there would be an issue ... so long as the AJAX add-to-cart implementation recognizes that it doesn't need to be active during checkout-related pages.
